app/assets/stylesheets/application.scss

```diff
@@ -30,7 +30,18 @@
  //   -ms-transform: translateY(-50%);
  //   transform: translateY(-50%);
  // }
+ .post-it {
+ background:#fefabc;
+ padding:15px;
+ font-size:15px;
+ color: #000;
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.